test/Directory.Build.props

```diff
@@ -2,9 +2,9 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) >= 16.8.0">$(TargetFrameworks);net5.0</TargetFrameworks>
+    <!-- only run .NET Framework tests on Windows -->
+    <TargetFrameworks Condition="'$(TargetFrameworks)' == '' and '$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
`'$(TargetFrameworks)' == ''` will this ever be the case? can't we just have the OS check here?

Reply:
No, we need to keep the `'$(TargetFrameworks)' == ''` check so that we can specify a smaller subset of target frameworks in project .csproj's, like when we can't build against `net452`

Reply:
I assumed this was here in case we specified `TargetFrameworks` externally, like a msbuild property in the command line. This file is included before the project file, and the project file overwrites any previous value of `<TargetFrameworks>`.

For example, `/src/Datadog.Trace.AspNet/Datadog.Trace.AspNet.csproj` overwrites the value from `/src/Directory.Build.props`.

https://github.com/DataDog/dd-trace-dotnet/blob/13ee2e4ed79dfa9332b2aa8a916899d2d5b43a01/src/Datadog.Trace.AspNet/Datadog.Trace.AspNet.csproj#L4

https://github.com/DataDog/dd-trace-dotnet/blob/13ee2e4ed79dfa9332b2aa8a916899d2d5b43a01/src/Directory.Build.props#L5